sympy/functions/special/bsplines.py

```diff
@@ -254,6 +279,20 @@ def interpolating_spline(d, x, X, Y):
     Piecewise((7*x**3/117 + 7*x**2/117 - 131*x/117 + 2, (x >= -2) & (x <= 1)),
             (10*x**3/117 - 2*x**2/117 - 122*x/117 + 77/39, (x >= 1) & (x <= 4)))
 
+    Parameters
+    ==========
+
+    d : integer
+        Degree of Bspline strictly greater than equal to one
+
+    X : list of strictly increasing integer values
+        list of X coordinates through which the spline passes
+
+    Y : list of strictly increasing integer values
+        list of Y coordinates through which the spline passes
+
+    x : symbol
```

Comment:
Maybe the parameters should be in the same order as for the function

Reply:
Done.